src/ext/components/partial-dependence-output.coffee

```diff
@@ -40,6 +42,8 @@ H2O.PartialDependenceOutput = (_, _go, _result) ->
           g.from table
         )
 
+      section.isFrameShown = lift _isFrameShown, (value)-> value
```

Comment:
before there was one value. when the checkbox is selected, the value is updated.